analyzers/tests/SonarAnalyzer.UnitTest/Rules/SillyBitwiseOperationTest.cs

```diff
@@ -24,41 +24,41 @@
 namespace SonarAnalyzer.UnitTest.Rules
 {
     [TestClass]
-    public class SillyBitwiseOperationTest
+    public class UnnecessaryBitwiseOperationTest
```

Comment:
The name of the file is still `SillyBitwiseOperationTest`

Reply:
Well spot, fixed!